src/styles/_catalog.scss

```diff
@@ -1,22 +1,31 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  justify-content: center;
+  gap: 48px;
   margin: 0 auto;
 
+  @media (max-width: 487px) {
+    grid-template-columns: 200px;
```

Comment:
it is better to put frequent repetitive values into a variable, this is necessary in order to, for example, change the width of the card, and not to look for all the places where you set it, but to change it only in one place (change the value of the variable)